src/group/groupFragments.js

```diff
@@ -27,3 +27,15 @@ export const groupMembers = `
     }
   }
 `
+
+export const accountMembership = `
+  fragment accountMembership on GroupNode { 
+    accountMembership: memberships(user_Email_In: [$accountEmail]) {
```

Comment:
Question: can't this information be fetched on the `groupMembers` fragment? :thinking:

Reply:
@cacarrara Is this question related to if this should be the correct place for this fragment? If that is it, the answer is yes since this is a fragment that applies over the `GroupNode` if that your question can you add more context to it?

Reply:
It was more about if we could get the member role on `groupMembers` and avoid heaving 2 fragments that get the same thing (memberships).

Reply:
@cacarrara Ohh I see, let me check that

Reply:
@cacarrara I checked. So, I think we should keep this as a separate fragment for some reasons:
- If for each group we fetch we always get all memberships then we will have to loop over them to match it with the current logged in user, adding extra work on the client
- When we add pagination, filtering for members, or just when we use the totalCount instead of getting the complete list of members we can't be sure that the membership for the current logged in user will be in that list
- Not every time we fetch the group data we need to know the role of the logged in user

So, this is why I think keeping a separate fragment just to get the information of the logged user account membership in the group is useful. Let me know if that makes sense